lib/ButtonNew/styles/buttonBase.scss

```diff
@@ -1,5 +1,5 @@
 .button-base {
-  @apply border border-solid leading-4 rounded font-display font-semi-bold inline-flex items-center relative;
+  @apply border border-solid leading-4 rounded font-display font-semi-bold inline-flex items-center relative no-underline;
```

Comment:
For context to others, when we were adding `button-base` styling to an `<a>` tag, it was still inheriting the browsers `text-decoration: underline.` This resolves that issue. Can't think of a case where we'd want a button to also be underlined.